src/utilities/scroll.ts

```diff
@@ -1,25 +1,24 @@
 import { getOffset } from './offset';
 
-let locks = [];
+const locks = new Set();
 
 //
 // Prevents body scrolling. Keeps track of which elements requested a lock so multiple levels of locking are possible
 // without premature unlocking.
 //
-export function lockBodyScrolling(lockingEl: HTMLElement) {
-  if (lockingEl && !locks.includes(lockingEl)) {
-    locks.push(lockingEl);
-    document.body.classList.add('sl-scroll-lock');
-  }
+export function lockBodyScrolling(lockingEl: HTMLElement | null) {
+  if (!lockingEl) return
```

Comment:
Good catch. The ` | null` and the conditional can go away entirely since calling the method without an argument is invalid (and it doesn't occur anywhere in the library).

Reply:
Sounds good! Just noticed in the original code that it was checking for falsy values and figured I would be safe and maintain the functionality. I will make that change 👍